src/Components/EditProfileModal/EditProfileModal.jsx

```diff
@@ -0,0 +1,102 @@
+import { useUserContext } from "Context";
+import { useState } from "react";
+import * as styles from "./EditProfileModal.module.css";
+
+export const EditProfileModal = ({
+  firstName,
+  bio,
+  profilePhoto,
+  coverPhoto,
+  setModal,
+}) => {
+  const { editProfile } = useUserContext();
+  const [editedData, setEditedData] = useState({
+    firstName: firstName,
+    bio: bio,
+    profilePhoto: profilePhoto,
+    coverPhoto: coverPhoto,
+  });
+  // console.log(editedData);
```

Comment:
If this is not required, you can remove this commented code.

Reply:
Yes left out by mistake